src/freenas/usr/local/lib/middlewared_truenas/plugins/failover.py

```diff
@@ -1246,6 +1250,18 @@ async def sync_keys_to_remote_node(self, lock=True):
                     self.middleware.logger.error('Failed to sync keys with remote node: %s', str(e), exc_info=True)
                 else:
                     await self.middleware.call('alert.oneshot_delete', 'FailoverKeysSyncFailed', None)
+                try:
+                    kmip_keys = await self.middleware.call('kmip.kmip_memory_keys')
+                    await self.middleware.call(
+                        'failover.call_remote', 'kmip.update_memory_keys', [kmip_keys]
+                    )
+                except Exception as e:
+                    await self.middleware.call('alert.oneshot_create', 'FailoverKMIPKeysSyncFailed', None)
```

Comment:
Why don't we put `str(e)` to the exception body so user has a better understanding why does this fail.